packaging/examples/kafka/nodepools/README.md

```diff
@@ -0,0 +1,30 @@
+# Kafka Node Pool examples
+
+The examples in this directory show how you can use the Kafka node pool to set up your Kafka cluster.
+The Kafka node pools are currently protected by a feature gate `KafkaNodePools` which is in _alpha_ phase and is disabled by default.
+
+_NOTE: Feature gates in the _alpha_ stage should be considered as experimental / in development and should not be used in production._
+_Backward incompatible changes might be made to such feature gates or they might be simply removed._
+
+If you want to try out the `KafkaNodePools` feature gate and the examples from this directory, you have to enable the feature gate first.
+
+## Regular examples
+
+The [`kafka.yaml`](./kafka.yaml) example shows a regular Kafka cluster backed by ZooKeeper.
+This example file deploys a ZooKeeper cluster with 3 nodes and 2 different pools of Kafka brokers.
+Each of the pools has 3 brokers.
+The pools in the example use different storage configuration.
```

Comment:
The persistent storage allows us to show at least some of the differences. I aimed with the example files at this stage to show the new things you could do with the node pools rather than just rewrite the existing files with it. Once it is enabled by default for everyone, we can rewrite the existing files to show how to do it with it. But now, when you need to enable it as an alpha feature gate, I expect people will be more interested in examples that make some use of the node pools rather than rewrite of exiting examples using node pools.

Reply:
I see your point, my only concern is just about the amount of resources they would need just for kicking the tires to the node pools support.